token.go

```diff
@@ -516,6 +518,39 @@ func parseInfo(r *tdsBuffer) (res Error) {
 	return
 }
 
+// https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-tds/140e3348-da08-409a-b6c3-f0fc9cee2d6e
+func parseTableNames(r *tdsBuffer) (tables []tableStruct) {
+	length := r.uint16()
+	_ = length // ignore length
```

Comment:
I suppose you could replace this with, assuming you need to call r.uint16() to mutate the state of r?
```go
_ = r.uint16()?
```
or otherwise remove these lines

Reply:
I am sticking to the original author's coding style.